### ISSUE TITLE
Unable to update hyperkit driver: Error downloading checksum file: bad response code: 404

### ISSUE THREAD

**Issue by medyagh** [kubernetes/minikube#7140]
on the head on mac os:
first purge
```
medmac@~/workspace/minikube (master) $ ./out/minikube delete --all --purge
🔥  Successfully deleted all profiles
💀  Successfully purged minikube directory located at - [/Users/medmac/.minikube]
```
then 
```
medmac@~/workspace/minikube (master) $ ./out/minikube start
😄  minikube v1.9.0-beta.2 on Darwin 10.13.6
✨  Automatically selected the hyperkit driver. Other choices: docker, virtualbox, vmwarefusion
💾  Downloading driver docker-machine-driver-hyperkit:
❗  Unable to update hyperkit driver: download failed: https://github.com/kubernetes/minikube/releases/download/v1.9.0-beta.2/docker-machine-driver-hyperkit?checksum=file:https://github.com/kubernetes/minikube/releases/download/v1.9.0-beta.2/docker-machine-driver-hyperkit.sha256: invalid checksum: Error downloading checksum file: bad response code: 404
💿  Downloading VM boot image ...
    > minikube-v1.9.0-beta.2.iso.sha256: 65 B / 65 B [-------] 100.00% ? p/s 0s
    > minikube-v1.9.0-beta.2.iso: 173.76 MiB / 173.76 MiB  100.00% 22.08 MiB p/
💾  Downloading preloaded images tarball for k8s v1.18.0-rc.1 ...
    > preloaded-images-k8s-v2-v1.18.0-rc.1-docker-overlay2-amd64.tar.lz4: 542.8
```

error : 
```
❗  Unable to update hyperkit driver: download failed: https://github.com/kubernetes/minikube/releases/download/v1.9.0-beta.2/docker-machine-driver-hyperkit?checksum=file:https://github.com/kubernetes/minikube/releases/download/v1.9.0-beta.2/docker-machine-driver-hyperkit.sha256: invalid checksum: Error downloading checksum file: bad response code: 404
```

**Comment by medyagh** [comment]
CC: @tstromberg is that related to Download refactor ?

**Comment by medyagh** [comment]
Ah, nevermind, I believe this is because v1.9.0-beta.2 has not been released yet.
(we could still fix this logic, that if that latest doesnt exist, it should try to download latest possible driver)

**Comment by tstromberg** [comment]
Correct, this is because we were mid-release. We could certainly improve the experience, especially if the user already has a perfectly fine driver laying around.

**Comment by medyagh** [comment]
> Correct, this is because we were mid-release. We could certainly improve the experience, especially if the user already has a perfectly fine driver laying around.

Exactly . We rarely update the hyperkit driver, we could potentially put the latest driver Sha and check for the locally available Sha and see if we need to download

**Comment by amitmaurya001** [comment]
Faced same issue..any update?

 minikube-v1.9.0.iso.sha256: 65 B / 65 B [--------------] 100.00% ? p/s 0s
    > minikube-v1.9.0.iso: 174.93 MiB / 174.93 MiB [ 100.00% 1.17 MiB p/s 2m29s
💾  Downloading Kubernetes v1.18.0 preload ...
    > preloaded-images-k8s-v2-v1.18.0-docker-overlay2-amd64.tar.lz4: 542.91 MiB
🔥  Creating hyperkit VM (CPUs=2, Memory=2200MB, Disk=20000MB) ...
🤦  StartHost failed, but will try again: creating host: create: Error creating machine: Error in driver during machine creation: hyperkit crashed! command line:
  hyperkit loglevel=3 console=ttyS0 console=tty0 noembed nomodeset norestore waitusb=10 systemd.legacy_systemd_cgroup_controller=yes random.trust_cpu=on hw_rng_model=virtio base host=minikube
🔥  Deleting "minikube" in hyperkit ...
🔥  Creating hyperkit VM (CPUs=2, Memory=2200MB, Disk=20000MB) ...
❌  StartHost failed again: creating host: create: Error creating machine: Error in driver during machine creation: hyperkit crashed! command line:
  hyperkit loglevel=3 console=ttyS0 console=tty0 noembed nomodeset norestore waitusb=10 systemd.legacy_systemd_cgroup_controller=yes random.trust_cpu=on hw_rng_model=virtio base host=minikube
👉  Run: "minikube delete", then "minikube start --alsologtostderr -v=1" to try again with more logging

E0329 00:05:07.799189   41597 style.go:173] unable to parse "💣  Unable to start VM after repeated tries. Please try {{'minikube delete' if possible\n": template: 💣  Unable to start VM after repeated tries. Please try {{'minikube delete' if possible
:1: malformed character constant: 'minikube delete' - returning raw string.
💣  Unable to start VM after repeated tries. Please try {{'minikube delete' if possible
❌  Error: [HYPERKIT_CRASHED] creating host: create: Error creating machine: Error in driver during machine creation: hyperkit crashed! command line:
  hyperkit loglevel=3 console=ttyS0 console=tty0 noembed nomodeset norestore waitusb=10 systemd.legacy_systemd_cgroup_controller=yes random.trust_cpu=on hw_rng_model=virtio base host=minikube
💡  Suggestion: Hyperkit is broken. Upgrade to the latest hyperkit version and/or Docker for Desktop. Alternatively, you may choose an alternate --driver
⁉️   Related issues:
    ▪ https://github.com/kubernetes/minikube/issues/6079
    ▪ https://github.com/kubernetes/minikube/issues/5780

**Comment by tstromberg** [comment]
This was fixed with the v1.9.0 release.

@amitmaurya001 - I'll move your issue into a separate bug.

**Comment by amitmaurya001** [comment]
how can i update to the latest version..coz i used latest version i guess..
